### ISSUE TITLE
Fedora Silverblue upgrades via cockpit-ostree causes removal of all overlays

### ISSUE THREAD

**Issue by shdwchn10** [cockpit-project/cockpit-ostree#258]
I can reproduce it with Fedora Silverblue 35 and 36
![scr](https://user-images.githubusercontent.com/79763381/168474920-14bd4ce5-7970-4e11-bab6-769a8af66a8a.png)

Maybe related #255

**Comment by garrett** [comment]
@martinpitt: My guess is that cockpit-ostree deploys the commit when clicking the "update and reboot" button, which (I think?) is correct when a local commit with overlays exists, but incorrect when something hasn't been deployed yet (as it won't actually upgrade but just deploy the remote commit without local changes).

**Comment by shdwchn10** [comment]
Yes, I tested upgrade in VM and can confirm that it's only a visual bug. Overlays still persist after upgrade via cockpit-ostree
